apps/core/ninja_utils/types.py

```
@@ -1,16 +1,23 @@
+
+def _build_url(url: str, info: SerializationInfo[SerializationInfoDict]) -> str:
     if not url:
         return url
-    request: Request = info.context["request"]
+    request = info.context["request"]
     return request.build_absolute_uri(url)
 
 
 AbsoluteUrl = Annotated[str, PlainSerializer(_build_url, return_type=str)]
+
+__all__ = [AbsoluteUrl]
```

Ruff (0.15.14)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)